contracts/paratii/TcrPlaceholder.sol

```diff
@@ -0,0 +1,169 @@
+pragma solidity ^0.4.18;
+
+import "./ParatiiToken.sol";
+
+/// WARNING , NOT PRODUCTION READY, based on ADCHAIN Registry.
+/// THIS IS JUST A PLACEHOLDER..
+contract TcrPlaceholder {
+
+  // Events
+  // ---------
+
+  event _Application(string videoId, uint deposit);
+  event _NewVideoWhitelisted(string videoId);
+
+  struct Listing {
+    uint applicationExpiry; // Expiration date of apply stage
+    bool whitelisted;       // registration status
+    address owner;          // Owner of the video.
+    uint unstakedDeposit;   // unlocked tokens
```

Comment:
Why *un*locked?

Reply:
Ah, I see now the reference implementation here: https://github.com/skmgoldin/tcr/blob/master/contracts/Registry.sol .. It has a better comment though :-)